src/features/side_bar/components/SideBar/SideBar.module.css

```diff
@@ -1,15 +1,18 @@
 .sideBarContainer {
   z-index: 1;
   display: inline-block;
-  width: 48px;
-  text-align: center;
-  margin: var(--unit) auto;
+  /* width: 48px; */
+  /* text-align: center; */
+  padding: 0 var(--unit);
+  margin: var(--unit) 0;
+  text-decoration: none;
```

Comment:
text-decoration can be removed as it's used from <a>

Reply:
it's needed! else the text becomes underlined because of routers <Link> component

Reply:
<img width="1355" alt="Screenshot 2022-08-26 at 09 03 12" src="https://user-images.githubusercontent.com/20676525/186833126-e1ec3aae-ca9b-4293-a656-88c5d2b4b8b9.png">
<img width="1312" alt="Screenshot 2022-08-26 at 09 03 34" src="https://user-images.githubusercontent.com/20676525/186833135-bfe0b42a-4949-4453-aa5d-f8d21caaa48a.png">

Reply:
I still have underline without that style
![image](https://user-images.githubusercontent.com/50058726/186854187-a3120e35-f77d-4ae8-ac5a-60642b5a28c0.png)
This is what brings it
![image](https://user-images.githubusercontent.com/50058726/186854424-65f2115f-415a-4eff-aa9a-e099385601f1.png)
What tag are you talking about?

Reply:
<img width="752" alt="Screenshot 2022-08-26 at 11 26 37" src="https://user-images.githubusercontent.com/20676525/186858366-01a5ddf0-fb87-4033-93ad-371960a48d5e.png">

Reply:
so as you see style from 'a' tag is applied
anyway it's not that major, lets stay as it is, if you see underline

Reply:
@IlyaIzr I've found that one extension in my browser affect this style, sorry for this.